vscode-antimony/src/server/stibium/stibium/analysis.py

```diff
@@ -225,6 +231,8 @@ def check_parse_tree(self, root, scope):
                     self.process_is_assignment(node, scope)
                 elif type(node.get_stmt()) == Assignment:
                     self.process_maybein(node, scope)
+                elif type(node.get_stmt()) == FluxBalanceConstraints:
+                    self.process_flux_balance(node, scope)
 
     def check_parse_tree_function(self, function, scope):

```

Comment:
either use typing or have a function comment that provides the type of arguments and return values (if not None).